userguide/shell.rst

```diff
@@ -40,7 +40,12 @@ desired, then press :kbd:`Enter` to re-enter the command.
 
 The :kbd:`Home`, :kbd:`End`, and :kbd:`Delete` are supported. :kbd:`Tab`
 completion is also available. Type a few letters and press :kbd:`Tab` to
-complete a command name or filename in the current directory.
+complete a command name or filename in the current directory. A helpful
+message can be displayed by right clicking inside of the shell. The
+message says *Context menu copy and paste operations are disabled in
```

Comment:
For this bit, it would be better to communicate the idea of what the help message is and maybe the commands rather than duplicating the message completely. Then we're immune from minor wording changes and will only need to update docs if the message changes from being about copy/paste or if the specific commands change:

"Right-clicking in the terminal window displays a helpful reminder about using :kbd:`Command+c` and :kbd:`Command+v` or :kbd:`Ctrl+Insert` and :kbd:`Shift+Insert` for copy and paste operations in the %brand% shell."